### PR TITLE
workshop-freeipa-versions

### PR DESCRIPTION
**Pull request by tscherf**
Add FreeIPA version requirements into unit 11 and unit 12.

### CODE REVIEW

File: doc/workshop/11-kerberos-ticket-policy.rst [hunk 1]
```diff
@@ -8,6 +8,8 @@ Unit 11: Kerberos ticket policy
 In this module you will explore how to manage properties of Kerberos
 authentication and authorization in FreeIPA.
 
+**Note:** To complete this module, FreeIPA-4.8.3 or later is needed.
+
```
[the remark this answers is not in the document]
Comment (tscherf) [reply doc/workshop/11-kerberos-ticket-policy.rst:12]: It seems it was backported into 4.8 branch https://github.com/freeipa/freeipa/commit/14ff82f3b86ceeb0836b0354b7edb965db72b494

[the remark this answers is not in the document]
Comment (rcritten) [reply doc/workshop/11-kerberos-ticket-policy.rst:12]: Drat, forgot to look in the ipa-4-8 branch for it. Yes, it's there in tag release-4-8-4+

[the remark this answers is not in the document]
Comment (tscherf) [reply doc/workshop/11-kerberos-ticket-policy.rst:12]: Ah, right, the earliest tag for this commit says 4-8.4. Is the following statement from the same unit still true in this case? 

""""
With FreeIPA 4.8.3 or later, Kerberos ticket policy allows an administrator to set different life time and renewal age for ticket granting tickets obtained with different pre-authentication methods. Each policy setting may include authentication indicator to say that the life time or renewal age applies to TGTs with which include this indicator.
"""
@abbra @rcritten

[the remark this answers is not in the document]
Comment (abbra) [reply doc/workshop/11-kerberos-ticket-policy.rst:12]: If it was released in 4.8.4, then please say so. 4.8.3 seems to be a bit optimistic.

**Comment by rcritten** [comment]
LGTM. I assume we need a backport to ipa-4-9. Is that sufficient?

**Comment by tscherf** [comment]
I think a backport to ipa-4.9 is good enough. Thanks.

**Comment by abbra** [comment]
master:

* 39354a47b923e07370e17d81288b18cf66ad8137 workshop: add freeipa version requirements
* aa6db128a7bed74d7fb478f7c476b45694593181 workshop: add freeipa version requirements

**Pull request closed without merging**